Misc/NEWS.d/next/Security/2018-09-11-18-30-55.bpo-17239.kOpwK2.rst

```diff
@@ -0,0 +1,3 @@
+The sax parser no longer processes external entities by default. External
```

Comment:
I suggest "The XML save parser". It's not obvious that sax is related to XML if you are not used to XML.